pkg/cloudprovider/suite_test.go

```diff
@@ -280,6 +282,23 @@ var _ = Describe("CloudProvider", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(drifted).To(Equal(ImageDrift))
 			})
+
+			It("should NOT trigger drift if node doesn't have kubelet client ID label", func() {
```

Comment:
nit: I was liking breaking the drift tests into `Context` based on their drift type.

Reply:
Yea, realizing this is in the `Node Image Drift` Context. Would be great to split it into its own `Context("Kubelet Identity Drift"`

Reply:
Oh I msised that in the rebase, can fix.